src/commands/tokens/send_ft/mod.rs

```diff
@@ -57,3 +64,113 @@ impl SendFtCommand {
+    let args = serde_json::to_vec(&json!({
+        "receiver_id": amount_ft.amount().to_string(),
+        "amount": amount_ft.amount().to_string(),
+        "memo": memo.as_ref().and_then(|s| {
+            let trimmed = s.trim();
+            if trimmed.is_empty() {
+                None
+            } else {
+                Some(trimmed.to_string())
+            }
+        })
+    }))?;
+
+    let action_ft_transfer = near_primitives::transaction::Action::FunctionCall(Box::new(
+        near_primitives::transaction::FunctionCallAction {
+            method_name: "ft_transfer".to_string(),
+            args,
+            gas: gas.as_gas(),
+            deposit: deposit.as_yoctonear(),
+        },
+    ));
+
+    let args = serde_json::to_vec(&json!({"account_id": receiver_account_id.to_string()}))?;
+
+    let call_result = network_config
+        .json_rpc_client()
+        .blocking_call_view_function(
+            ft_contract_account_id,
+            "storage_balance_of",
+            args.clone(),
+            near_primitives::types::Finality::Final.into(),
+        )
+        .wrap_err_with(||{
+            format!("Failed to fetch query for view method: 'storage_balance_of' (contract <{}> on network <{}>)",
+                ft_contract_account_id,
+                network_config.network_name
+            )
+        })?;
+
+    if call_result.parse_result_from_json::<Value>()?.is_null() {
+        let action_storage_deposit = near_primitives::transaction::Action::FunctionCall(Box::new(
+            near_primitives::transaction::FunctionCallAction {
+                method_name: "storage_deposit".to_string(),
+                args,
+                gas: gas.as_gas(),
+                deposit: near_token::NearToken::from_millinear(100).as_yoctonear(),
+            },
+        ));
+        return Ok(crate::commands::PrepopulatedTransaction {
+            signer_id: signer_id.clone(),
+            receiver_id: ft_contract_account_id.clone(),
+            actions: vec![action_storage_deposit, action_ft_transfer.clone()],
+        });
+    }
+
+    Ok(crate::commands::PrepopulatedTransaction {
+        signer_id: signer_id.clone(),
+        receiver_id: ft_contract_account_id.clone(),
+        actions: vec![action_ft_transfer.clone()],
+    })
+}
+
+pub fn get_amount_ft(
+    ft_transfer_amount: &crate::types::ft_properties::FungibleTokenTransferAmount,
+    network_config: &crate::config::NetworkConfig,
+    signer_account_id: &near_primitives::types::AccountId,
+    ft_contract_account_id: &near_primitives::types::AccountId,
+) -> color_eyre::eyre::Result<crate::types::ft_properties::FungibleToken> {
+    match ft_transfer_amount {
+        crate::types::ft_properties::FungibleTokenTransferAmount::ExactAmount(ft) => Ok(ft.clone()),
+        crate::types::ft_properties::FungibleTokenTransferAmount::MaxAmount => {
+            let function_args =
+                serde_json::to_vec(&json!({"account_id": signer_account_id.to_string()}))?;
```

Comment:
Try dropping `.to_string()`, it should automatically use that anyway, but it will be easier to read without it.

```suggestion
                serde_json::to_vec(&json!({"account_id": signer_account_id}))?;
```